styles.css

```diff
@@ -249,3 +249,94 @@ hr {
 	color: #dd1972;
   margin: -20px;
 }
+
+.winner-card{
+display: flex;
+flex-direction: column;
+align-items: center;
+height: 250px;
+background-color: white;
+margin-top: 20px;
+margin-left: 40px;
+margin-right: 40px;
+
+}
+
+.win-card-header{
+	display: flex;
+	flex-direction: row;
+	height: 55px;
+	border-bottom: solid #d0d2d3 1px
+}
+
+.vs-span{
+	mergin-left: 40px;
+	margin-right: 40px;
+}
+
+.win-top-row{
+	margin: 20px;
+	color: #404041;
+
+}
+.name-span{
+	 font-weight: 700;
+	 font-size: 20px
+ }
+
+.vs-span{
```

Comment:
The formatting of everything looks close enough to give us something to build off we can work on the specifics later.  The only thing I saw that we will need to remember to add is the CSS for the span that sits around the range numbers display and also the span that adds the underline to the cards in the right column.